### ISSUE TITLE
mkvirtualenv -h ends with 'The system cannot find the path specified.'

### ISSUE THREAD

**Issue by jay** [davidmarble/virtualenvwrapper-win#68]
I'm using virtualenv 15.0.3 and I installed virtualenvwrapper-win from master 8a491f0 2016-01-24. When I run `mkvirtualenv -h` it shows me the options but then ends with 

```
...
  --distribute          DEPRECATED. Retained only for backward compatibility.
                        This option has no effect.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
```

[mkvirtualenv.bat line 59](https://github.com/davidmarble/virtualenvwrapper-win/blob/8a491f0/scripts/mkvirtualenv.bat#L59) says 

``` batch
>>"%WORKON_HOME%\%ENVNAME%\Scripts\activate.bat" (
```

However since ENVNAME comes from %~1 it expands to `"C:\Users\Username\Envs\-h\Scripts\activate.bat"` I'd guess the other two errors are from the same thing.

**Comment by thebjorn** [comment]
Verified:
```
(foo) c:\srv> mkvirtualenv -h
Usage: virtualenv [OPTIONS] DEST_DIR

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -v, --verbose         Increase verbosity.
  -q, --quiet           Decrease verbosity.
  -p PYTHON_EXE, --python=PYTHON_EXE
                        The Python interpreter to use, e.g.,
                        --python=python2.5 will use the python2.5 interpreter
                        to create the new environment.  The default is the
                        interpreter that virtualenv was installed with
                        (c:\python27\python.exe)
  --clear               Clear out the non-root install and start from scratch.
  --no-site-packages    DEPRECATED. Retained only for backward compatibility.
                        Not having access to global site-packages is now the
                        default behavior.
  --system-site-packages
                        Give the virtual environment access to the global
                        site-packages.
  --always-copy         Always copy files rather than symlinking.
  --unzip-setuptools    Unzip Setuptools when installing it.
  --relocatable         Make an EXISTING virtualenv environment relocatable.
                        This fixes up scripts and makes all .pth files
                        relative.
  --no-setuptools       Do not install setuptools (or pip) in the new
                        virtualenv.
  --no-pip              Do not install pip in the new virtualenv.
  --extra-search-dir=DIR
                        Directory to look for setuptools/pip distributions in.
                        This option can be used multiple times.
  --never-download      DEPRECATED. Retained only for backward compatibility.
                        This option has no effect. Virtualenv never downloads
                        pip or setuptools.
  --prompt=PROMPT       Provides an alternative prompt prefix for this
                        environment.
  --setuptools          DEPRECATED. Retained only for backward compatibility.
                        This option has no effect.
  --distribute          DEPRECATED. Retained only for backward compatibility.
                        This option has no effect.
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.

c:\srv>
```

**Comment by thebjorn** [comment]
This is fixed in PR #82.
